dev-util/bpftrace/bpftrace-0.14.0-r3.ebuild

```diff
@@ -70,19 +62,23 @@ pkg_pretend() {
 }
 
 pkg_setup() {
-	LLVM_MAX_SLOT=12 llvm_pkg_setup
-}
-
-src_prepare() {
-	cmake_src_prepare
+	LLVM_MAX_SLOT=13 llvm_pkg_setup
 }
 
 src_configure() {
 	local -a mycmakeargs=(
 		-DSTATIC_LINKING:BOOL=OFF
+		-DBUILD_SHARED_LIBS:=OFF
 		-DBUILD_TESTING:BOOL=OFF
 		-DBUILD_FUZZ:BOOL=$(usex fuzzing)
+		-DENABLE_MAN:BOOL=OFF
```

Comment:
any reason?

Reply:
Hmm I'll check if we can switch it to on (this was same as in previous versions, but GH interface showed it as being added in this revision)

Reply:
Ah, ok, don't treat it as a blocker for merging then given some of these fixes are quite important

Reply:
okay